src/measure_theory/borel_space.lean

```diff
@@ -763,10 +918,30 @@ L.measurable.comp φ_meas
 
 end continuous_linear_map
 
+section normed_space
+variables {E : Type*} [measurable_space E] [normed_group E]
+variables [second_countable_topology E] [normed_space ℝ E] [borel_space E]
+
+/-- Currently we only prove this lemma with `ℝ` as the base field, and use Hahn-Banach in the proof.
+ In the future we might be able to generalize the statement and give a more elementary proof. -/
```

Comment:
update docstring?

Reply:
Yes, good point. I'm quite certain the current build is broken anyway, so I'm going to do a second pass on it after the build finishes.